### ISSUE TITLE
Allow buttons to receive user keyboard focus

### ISSUE THREAD

**Issue by ubercj** [e-gineering/movement-challenge-calculator#3]
https://github.com/e-gineering/movement-challenge-calculator/blob/638acc8f6179bd9a9457b26b86f705255f35f961/index.html#L103-L110

This button is one example where a `tabindex` of -1 is set on a button. This prevents a user from being able to use these buttons entirely unless they are using a mouse. This could possible violate [WCAG SC 2.1.1 Keyboard](https://www.w3.org/TR/wcag2mobile-22/#success-criterion-2-1-1-keyboard)
